src/library/autodj/dlgautodj.cpp

```diff
@@ -270,29 +275,30 @@ void DlgAutoDJ::fadeNowButton(bool) {
 }
 
 void DlgAutoDJ::toggleAutoDJButton(bool enable) {
-    AutoDJProcessor::AutoDJError error = m_pAutoDJProcessor->toggleAutoDJ(enable);
+    m_pAutoDJProcessor->toggleAutoDJ(enable);
+}
+
+// TODO If there's a way to migrate the translations move this
+// to AutoDJProcessor in order to keep this class minimal
+void DlgAutoDJ::autoDJError(AutoDJProcessor::AutoDJError error) {
```

Comment:
I have no experience with Transifex: is it smart enough to keep the existing translations?

Reply:
Yes, as long as the original string and the name of the surrounding QObject is not changed.